### DISCUSSION TITLE
Should we mark all the find methods with @discardableResult?

### DISCUSSION THREAD

**Discussion by josh-arnold-1** [nalexn/ViewInspector#355] [General]
A lot of the time I find myself wanting to try to find a view, just to assert on its existence, so I end up writing:

```swift
_ = sut.find(SomeView.self) { ... }
```

At the moment, to avoid a warning, we need to specify a placeholder here `_`.

Would we be against marking the find methods with `@discardableResult` so we can use `find` almost as an assertion in itself?

Thanks!

**Comment by JOyo246** [comment]
Agreed. But duplicate of https://github.com/nalexn/ViewInspector/issues/349
